docs/usage.md

```diff
@@ -20,12 +20,11 @@ What follows are the specific steps for each of these tabs.
 
 ### 1. Git operations
 
-From this tab you can do Git operations,
-such as checking out our development branches:
+From this tab you can do Git operations, such as checking out our master
+branch:
```

Comment:
This is explicitly the kind of change that semantic line breaks hope to avoid ;)

Leave it broken at the comma, and the diff here only affects L24.